base_app/package.json

```diff
@@ -10,80 +10,51 @@
   "author": "Jim Scarsdale",
   "license": "ISC",
   "scripts": {
-    "ng": "./node_modules/.bin/ng",
+    "ng": "node node_modules/@angular/cli/bin/ng.js",
     "build-client-dev": "bash make-version-txt.bash build-client-only && rm -rf dist dist.tgz && ng build --output-hashing all --configuration development && bash post-build-step.bash --include-mocks",
-    "build-client-prod": "bash make-version-txt.bash build-client-prod && rm -rf dist dist.tgz && ng build --aot --output-hashing all --no-source-map --no-named-chunks && bash post-build-step.bash --include-mocks",
-    "start-client-dev": "ng serve",
-    "start-client-prod": "ng serve --aot",
+    "build-client-prod": "bash -c \"bash make-version-txt.bash build-client-prod && rm -rf dist dist.tgz && ng build --aot --output-hashing all --no-source-map --no-named-chunks && bash post-build-step.bash\"",
+    "start-client-dev": "cd ../ && bash run-dev-env.bash",
+    "start-client-prod": "bash -c \"node node_modules/@angular/cli/bin/ng.js serve --aot\"",
     "post-build-step--setup-bms-for-dev": "sh build-app-for-dev.sh 0 bms git-workspace-all ${USER}",
-    "test": "ng test",
-    "test-headless-once": "ng test --browsers ChromeHeadless --watch=false",
-    "lint--not-tested": "ng lint",
-    "e2e--not-tested": "ng e2e"
+    "test": "node node_modules/@angular/cli/bin/ng.js test --browsers Chrome_small",
+    "test-headless-once": "node node_modules/@angular/cli/bin/ng.js test --browsers ChromeHeadlessWin --watch=false",
+    "lint--not-tested": "bash -c \"node node_modules/@angular/cli/bin/ng.js lint\"",
+    "e2e--not-tested": "bash -c \"node node_modules/@angular/cli/bin/ng.js e2e\""
```

Comment:
We should update to current angular guidelines (eslint or whatever), test, and rename removing "--not-tested".

Reply:
installed eslint via the angular cli tested it out. Works well! `ng lint`